app/assets/stylesheets/common/base/search-menu.scss

```diff
@@ -327,8 +331,10 @@ $search-pad-horizontal: 0.5em;
       padding: 0px 3px;
       display: inline-block;
       background-color: transparent;
+      .btn.search-context {
+      }
```

Comment:
Can we remove this empty rule-set?

Reply:
wow, not sure how that made it lol

Reply:
yeah, shouldn't this be caught by linting in CI? 🤔 @CvX ?